data/brandColors.js

```diff
@@ -757,6 +758,10 @@ var _brandColors = {
   'product-hunt-4': '#00b27f',
   'pusha': '#0f71b4',
   'qantas-airlines': '#e0001b',
+  'qq-1': '#000000',
```

Comment:
Just to follow the current (very stupid) semantic that I chose to use, can you please remove the "-1" on the first color? :)